dipy/utils/arrfuncs.py

```diff
@@ -28,3 +28,45 @@ def as_native_array(arr):
         return arr
     return arr.byteswap().newbyteorder()
 
+
+def pinv_vec(a, rcond=1e-15):
+    """Vectorized version of numpy.linalg.pinv"""
```

Comment:
Documentation should follow the numpy docstring standard: https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt